src/main/kotlin/uk/gov/justice/digital/hmpps/prisonregister/model/PrisonFilter.kt

```diff
@@ -9,14 +9,18 @@ import javax.persistence.criteria.Root
 
 class PrisonFilter(
   val active: Boolean? = null,
-  val textSearch: String? = null
+  val textSearch: String? = null,
+  val genders: List<Gender>? = listOf(),
 ) : Specification<Prison> {
 
   override fun toPredicate(root: Root<Prison>, query: CriteriaQuery<*>, cb: CriteriaBuilder): Predicate? {
     val andBuilder = ImmutableList.builder<Predicate>()
     active?.let {
       andBuilder.add(cb.equal(root.get<Any>("active"), it))
     }
+    genders?.forEach {
+      andBuilder.add(cb.equal(root.get<Any>(it.columnName), true))
```

Comment:
Thought about using `it.name.lowercase()` but thought having a field on the enum was clearer